modules/material/public/styles.2.3.6.css

```diff
@@ -1,3 +1,7 @@
+body {
+  display: grid;
```

Comment:
CSS is so mysterious. How this achieves the effect you describe is not easy to understand. I'm just editorializing, not asking for an explanation necessarily.

Reply:
Maybe we should let ai do our css coding more and just treat it like magic.

Reply:
I've heard it said that AI shouldn't have to create readable code...probably machine language would be best. It only does that for us stupid humans.